tests/10-vcdm2.js

```diff
@@ -11,6 +11,7 @@ import {createRequire} from 'module';
 import {filterByTag} from 'vc-test-suite-implementations';
 import {TestEndpoints} from './TestEndpoints.js';
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
This is a chore not suitable to this PR, but we really should figure out where chai.should should live and and an eslint rule for the `.eslintrc.js` file that allows this. I think technically speaking we should init should once before the tests run at all. There's a mocha file for it somewhere.